client/resource_feature_store.go

```diff
@@ -105,6 +105,11 @@ func destinationSchema() *schema.Resource {
 				Optional:     true,
 				ValidateFunc: validation.StringIsNotWhiteSpace,
 			},
+			"topic": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: validation.StringIsNotWhiteSpace,
+			},
```

Comment:
Not your fault, but there should be a oneOf on one of these.

Reply:
Added above but it just fails:

```
│ Error: InternalValidate
│
│   with provider["registry.terraform.io/simple-machines/anaml"],
│   on main.tf line 16, in provider "anaml":
│   16: provider "anaml" {
│
│ Internal validation of the provider failed! This is always a bug
│ with the provider itself, and not a user issue. Please report
│ this bug:
│
│ 1 error occurred:
│ 	* resource anaml_table: ExactlyOneOf: folder references unknown attribute (folder) at part (folder)
```